sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -801,13 +809,52 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
             webpackResourceFiles,
             entries,
             targetDir,
+            stage.value.data,
             log
           ).to[Set]
         }
 
       cachedActionFunction(monitoredFiles.to[Set]).to[Seq]
     }
 
+  def entryPointTask(
+      stage: TaskKey[Attributed[File]]): Def.Initialize[Task[File]] =
+    Def.taskDyn {
+      val linkerConfig = (scalaJSLinkerConfig in stage).value
+      val linker = (scalaJSLinker in stage).value
+      val linkerTag = (usesScalaJSLinkerTag in stage).value
+      Def
+        .task {
+          val targetDir = (crossTarget in stage).value
+          val logger = streams.value.log
```

Comment:
Thanks for this. I'll make sure I understand it and update the PR with the fix in a bit. #